projects/dotcms/src/lib/models/DotCMSContent.model.ts

```diff
@@ -1,3 +1,11 @@
 export interface DotCMSContent {
     stName: string;
 }
+
+export interface DotCMSContentQuery {
```

Comment:
```typescript
export interface DotCMSContentQuery {
    query: string;
    options: {
        depth?: string;
        limit?: string;
        offset?: string;
        orderBy?: string;
    }
}
```

If you do it this way then here you can do `populateQueryUrl(params.options)` and you don't have to do the logic of avoid the property `query`